typos-ranking-rules.md

```diff
@@ -0,0 +1,59 @@
+# Typo and Ranking rules
+
+This is an explanation of the default rules used in MeiliDB.
+
+First of all we must explain some terms that are used in this reading.
+
+- A query string is the full list of all the words that the end user is searching for results.
+- A query word is one of the word that composes the query string.
+
+
+
+## Typo rules
+
+The typo rules are used before sorting the documents, they are used to aggregate them, to choose which documents contains words similar to the queried words.
```

Comment:
The typo rules are used before sorting the documents. They are used [...] documents _contain_ words [...]